### PR TITLE
Renames sequence events to actions

### LINKED ISSUE #12004: Rename sequence events to actions

### PR DESCRIPTION
**Pull request by bkase**
if it compiles, it runs

fixes #12004

### CODE REVIEW

**Comment by bkase** [comment]
!ci-build-me

**Review by rbonichon — APPROVED** [review]
No issue here. Looks like you need to run `ocamlformat` on (some of) the changed files to appease the CI

**Comment by bkase** [comment]
!ci-build-me

**Comment by bkase** [comment]
thanks for being my compiler while I was having trouble with my dev env, @MartinMinkov !

**Review by MartinMinkov — APPROVED** [review]

File: src/lib/transaction_logic/zkapp_command_logic.ml [hunk 1]
```diff
@@ -975,12 +975,12 @@ module Make (Inputs : Inputs_intf) = struct
...
 
-  let update_sequence_state (sequence_state : _ Pickles_types.Vector.t)
-      sequence_events ~txn_global_slot ~last_sequence_slot =
+  let update_sequence_state (sequence_state : _ Pickles_types.Vector.t) actions
+      ~txn_global_slot ~last_sequence_slot =
     (* Push events to s1. *)
     let [ s1'; s2'; s3'; s4'; s5' ] = sequence_state in
-    let is_empty = Sequence_events.is_empty sequence_events in
-    let s1_updated = Sequence_events.push_events s1' sequence_events in
+    let is_empty = Sequence_events.is_empty actions in
```
Comment (mrmr1993) [inline src/lib/transaction_logic/zkapp_command_logic.ml:982]: Do you want to rename the `Sequence_events` module to `Actions` (or maybe `Zkapp_actions`?

**Review by mrmr1993 — APPROVED** [review]

File: src/lib/transaction_logic/zkapp_command_logic.ml (hunk 1 above)
Comment (bkase) [reply src/lib/transaction_logic/zkapp_command_logic.ml:982]: shoot missed this one

**Comment by bkase** [comment]
!ci-build-me

**Comment by bkase** [comment]
!ci-build-me

**Comment by Trivo25** [comment]
!ci-build-me

**Comment by bkase** [comment]
!ci-build-me

**Comment by mitschabaude** [comment]
!ci-build-me

**Pull request merged**